Classes/People/PeopleViewController.swift

```diff
@@ -90,17 +109,16 @@ PeopleSectionControllerDelegate {
             }
         }
     }
-    
+
     // MARK: Private API
 
-    func updateSelectionCount() {
-        let label = UILabel()
-        label.font = Styles.Text.body.preferredFont
-        label.backgroundColor = .clear
-        label.textColor = Styles.Colors.Gray.light.color
-        label.text = "\(selected.count)/\(selectionLimit)"
-        label.sizeToFit()
-        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: label)
+    private func updateTitle() {
+        let selectedCount = "\(selected.count)/\(selectionLimit)"
+        switch type {
+        case .assignee: title = "\(NSLocalizedString("Assignees", comment: "")) \(selectedCount)"
+        case .reviewer: title = "\(NSLocalizedString("Reviewers", comment: "")) \(selectedCount)"
```

Comment:
I think we should keep the selection count formatting b/c there is a maximum for selected people (10)